src/map/map.cpp

```diff
@@ -785,7 +792,13 @@ int32 parse(int8* buff, size_t* buffsize, sockaddr_in* from, map_session_data_t*
             {
                 ShowWarning("[PacketGuard] Caught mismatch between player substate and recieved packet: Player: %s - Packet: %03hX",
                             PChar->getName(), SmallPD_Type);
-                // TODO: Plug in optional jailutils usage
+
+                if (settings::get<bool>("map.PACKETGUARD_DROP_CONNECTION"))
+                {
+                    ShowWarning("[PacketGuard] Removing %s's session from the server", PChar->getName());
```

Comment:
super nit: this isn't actually removing their session but making their packets ignored, if we wanted to actually remove it we could drop their session entry in sql so the key no longer works (but that would also spam the server with pointless "ignoring connection from...")

That might allow them to relog immediately, though...